scarlet/initialisation.py

```diff
@@ -0,0 +1,136 @@
+from .source import *
+from .interpolation import interpolate_observation
+from .observation import Observation
+from .wavelet import Starlet
+
+def build_initialisation_coadd(observations, wavelet = False):
+    """Build a channel weighted coadd to use for source detection
+
+    Parameters
+    ----------
+    observations: `~scarlet.observation.Observation`
+        Observation to use for the coadd.
+
+    Returns
+    -------
+    detect: array
+        2D image created by weighting all of the channels by SED
+    bg_cutoff: float
+        The minimum value in `detect` to include in detection.
+    """
+    try:
+        iter(observations)
+    except TypeError:
+        observations = [observations]
+    # The observation that lives in the same plane as the frame
+    loc = np.where([type(obs) is Observation for obs in observations])
+    # If more than one element is an `Observation`, then pick the first one as a reference (arbitrary)
+    obs_ref = observations[loc[0][0]]
```

Comment:
That needs to be at least explained to the user. We used to have the `obs_idx` keyword exposed to the user, but this is completely implicit. I'm also not sure how this would work with a joint observation coadd

Reply:
This is tricky. Actually, I put that here only because the current code suggests that we might feed multiple `Observation`s but I hardly a case where one would need to fit more than one `Observation` and here I make the distinction between `Observation` and `LowResObservation`. I think we should expect lists that don't have more than one `Observation`.

Reply:
I can think of a few counter examples, for example multi-epoch observations, where there is a separate `Observation` for each epoch. So I prefer to require to set the observation index if the length of `observations` is greater than 1, if for no other reason than to make the user think about what they are doing.

Reply:
Won't these have to be `LowResObservation`s? If it requires interpolation they will.

Reply:
Solved with an optional obs_idx. None leads to selecting the first one, because if you care which frame you are fitting into, you know what you are doing and otherwise, users probably won't care or hav only one `Observation`.